src/pretty-shadow-dom.ts

```diff
@@ -1,6 +1,17 @@
 import { prettyDOM, getConfig } from "@testing-library/dom";
 import type { Config, NewPlugin, Printer, Refs } from "pretty-format";
 
+// This regexp combo took way too long to figure out...
+const findWhiteSpace = /([^\S\r\n]*[\f\n\r\t\v]+)/.source;
+
+function removeDuplicateNewLines(str: string) {
+  let final = str.replace(
+    new RegExp(`${findWhiteSpace}.*${findWhiteSpace}{2,}`, "g"),
```

Comment:
Would this mess with a string like ` abc  `? (`<space>abc<space><space>`)? The `.*` part is what raised red flags in my head.

Reply:
It’s testing against line breaks + empty space first, in testing with template literals it doesn’t seem to affect it.